linux-admin/part-2-tasks/tasks_othman.md

```diff
@@ -0,0 +1,154 @@
+# First step on command line.
+1. Display your current directory.
+
+pwd
+
+2. Change to the /etc directory.
+
+cd /etc
+
+3. Now change to your home directory using only three key presses.
+
+cd
+
+4. Go to the parent directory of the current directory.
+
+cd ..
+
+5. Go to the root directory.
+
+cd /
+
+6. List a long listing of the root directory using two ways.
+
+ll 
+ls -l
+
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+
+ls -lhtr
```

Comment:
you need to also include a with the ls command in order to list the hidden files